packages/cms-lib/projects.js

```diff
@@ -175,8 +176,37 @@ async function createProject(dest, type, repoName, sourceDir, options = {}) {
   return success;
 }
 
+/**
+ * Writes a copy of the boilerplate project to dest.
+ * @param {String} dest - Dir to write project src to.
+ * @param {String} type - Type of project to create.
+ * @param {String} repoName - Name of GitHub repository to clone.
+ * @param {String} sourceDir - Directory in project that should get copied.
+ * @param {Object} options
+ * @returns {Boolean} `true` if successful, `false` otherwise.
+ */
+async function createVueProject(dest, type, repoName) {
+  const relativePath = path.relative(process.cwd(), dest);
+
+  return new Promise(resolve => {
+    const vueInit = spawn(
+      'vue',
```

Comment:
What if `vue` is not installed?

Reply:
Thanks! I had mean't to handle this -- I will look into it.